ml-agents/mlagents/trainers/ghost/controller.py

```diff
@@ -0,0 +1,92 @@
+from mlagents_envs.logging_util import get_logger
+from typing import Deque, Dict
+from collections import deque
+from mlagents.trainers.ghost.trainer import GhostTrainer
+
+logger = get_logger(__name__)
+
+
+class GhostController(object):
```

Comment:
```suggestion
class GhostController:
```
Not needed in python3